app/views/registers/in_progress.html.haml

```diff
@@ -42,37 +42,11 @@
               %th{role: "columnheader", scope: "col"}
                 Status
           %tbody
-            - @upcoming_registers.each_with_index do |register, index|
+            - @registers.each_with_index do |register, index|
               - index = index + 1
```

Comment:
seems like this `index` is only used for a `data-title` attribute in the template, can we just delete it or does it have a purpose?

Reply:
Removed 👍